models/aeif_psc_delta_clopath.h

```diff
@@ -0,0 +1,443 @@
+V_reset     double - Reset value for V_m after a spike. In mV.
+E_L         double - Leak reversal potential in mV.
+g_L         double - Leak conductance in nS.
+I_e         double - Constant external input current in pA.
+tau_plus    double - Time constant of u_bar_plus.
+tau_minus   double - Time constant of u_bar_minus.
+tau_bar_bar double - Time constant of u_bar_bar.
+
+Spike adaptation parameters:
+a          double - Subthreshold adaptation in nS.
+b          double - Spike-triggered adaptation in pA.
+Delta_T    double - Slope factor in mV.
+tau_w      double - Adaptation time constant in ms.
+V_peak     double - Spike detection threshold in mV.
+V_th_max   double - Value of V_th afer a spike in mV.
+V_th_rest  double - Resting value of V_th in mV.
+
+Clopath rule parameters:
+u_bar_plus    double - Low-pass filtered Membrane potential in mV.
+u_bar_minus   double - Low-pass filtered Membrane potential in mV.
+u_bar_bar     double - Low-pass filtered u_bar_minus in mV.
+A_LTD         double - Amplitude of depression in 1/mV.
+A_LTP         double - Amplitude of facilitation in 1/mV^2.
+theta_plus    double - threshold for u in mV.
+theta_minus   double - threshold for u_bar_[plus/minus] in mV.
+A_LTD_const   bool   - Flag that indicates whether A_LTD_ should
+                       be constant (true, default) or multiplied by
+                       u_bar_bar^2 / u_ref_squared (false).
+delay_u_bars  double - Delay with which u_bar_[plus/minus] are processed
+                       to compute the synaptic weights.
+U_ref_squared double - Reference value for u_bar_bar_^2.
+
+Other parameters:
+t_clamp      double - Duration of clamping of Membrane potential after a spike
+                      in ms.
+V_clamp      double - Value to which the Membrane potential is clamped in mV.
+
+Integration parameters:
+gsl_error_tol double - This parameter controls the admissible error of the
+                       GSL integrator. Reduce it if NEST complains about
+                       numerical instabilities.
+
+Note:
+
+Neither the clamping nor the delayed processing of u_bar_[plus/minus] are
+mentioned in [1]. However, they are part of an reference implementation
+by Claudia Clopath et al. that can be found on ModelDB [3]. The clamping is
+important to mimic a spike which is otherwise not described by the aeif neuron
+model.
+
+Author: Jonas Stapmanns, David Dahmen, Jan Hahne
+
+Sends: SpikeEvent
+
+Receives: SpikeEvent, CurrentEvent, DataLoggingRequest
+
+References:
+
+[1] Clopath et al. (2010) Connectivity reflects coding:
+    a model of voltage-based STDP with homeostasis.
+    Nature Neuroscience 13:3, 344--352
+[2] Clopath and Gerstner (2010) Voltage and spike timing interact
+    in STDP – a unified model. Front. Synaptic Neurosci. 2:25
+    doi: 10.3389/fnsyn.2010.00025
+[3] Voltage-based STDP synapse (Clopath et al. 2010) on ModelDB
+    https://senselab.med.yale.edu/ModelDB/showmodel.cshtml?model=144566
```

Comment:
On the other hand, the clamped terms appear only in https://senselab.med.yale.edu/ModelDB/showmodel.cshtml?model=144566&file=/modeldb_package/VoTriCode/aEIF.m if i see this correctly.
Maybe mention either file?

Reply:
You are right, we should be more precise. We added the link to the exact files: The first one to `hh_psc_alpha_clopath.h` and the second to `aeif_psc_delta_clopath.h`. The difference between the two links is that the first describes the synapse which is than connected with a Hodgkin Huxley neuron (see comment in <https://senselab.med.yale.edu/ModelDB/showmodel.cshtml?model=144566&file=%2fmodeldb_package%2fclaudia_pairing.py>). The HH neuron does not need clamping since it models the entire spike whereas the aeif neuron only models the upswing. The delay is present in both implementations. In the neuron code it is defined in `delay_steps = 50` and in the matlab implementation it is visible in `u_md_sig = ((u_md(t-3)-tetam) > 0)*(u_md(t-3)-tetam);` in <https://senselab.med.yale.edu/ModelDB/showmodel.cshtml?model=144566&file=%2fmodeldb_package%2fVoTriCode%2fVoTri.m>.